src/WingetCreateCLI/Properties/Resources.resx

```diff
@@ -540,7 +540,7 @@
     <value>More help can be found at: https://aka.ms/winget-create</value>
   </data>
   <data name="PrivacyStatement_HelpText" xml:space="preserve">
-    <value>Privacy statement: https://aka.ms/privacy</value>
+    <value>Privacy statement: https://aka.ms/winget-create-privacy</value>

```

Comment:
```suggestion
    <value>Privacy statement: {0}</value>
    <comment>{0} represents the privacy statement url</comment>
```

Make sure to use `string.Format` to add the new privacy url for all places that use this resource string.